src/sql/azdata.proposed.d.ts

```diff
@@ -1934,4 +1934,20 @@ declare module 'azdata' {
 			isPrimary: boolean;
 		}
 	}
+
+	export interface ModelBuilder {
+		doughnutChart(): ComponentBuilder<DoughnutChartComponent, DoughnutChartComponentProperties>;
+	}
+
+	export interface DoughnutChartComponentProperties extends ComponentProperties {
+		data: number[];
```

Comment:
Instead of having separate fields for each of these why not just have data be an array of objects that each contain a number, a label and an optional color? That seems much easier to use and less chance that numbers get mixed up.